varexport/src/test/java/com/indeed/util/varexport/VarExporterTest.java

```diff
@@ -679,7 +679,7 @@ public void testExportTwoLevelMap() {
         // NOTE: multi-level maps are not yet supported
         assertThat(
             Lists.newArrayList(out.toString().split("\n")),
-            Matchers.contains("map#nullvalue=null", "map#m1={2\\=two, 1\\=one}", "map#m2={two\\=2, one\\=1}"));
+            Matchers.contains("map#nullvalue=null", "map#m1={1\\=one, 2\\=two}", "map#m2={one\\=1, two\\=2}"));
```

Comment:
It passes without this change, right? Unless I misunderstand the point of Matchers.contains.